sound/soc/sof/sof-client.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause) */
+/*
```

Comment:
@ranj063 this patch doesn't contain the register/unregister method? Is it intended? Then you will need to update the commit message.